### PR TITLE
Feature add heading markdown

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket

https://trello.com/c/QAVgnXw4/287-user-story-ecosystem-actor-profile-page

# What solved
- [X] Should create a component for makerdao for the new description and sentence description

# Description
This adds a heading for the markdown component

### CODE REVIEW

File: src/stories/containers/ActorsAbout/ActorAboutContainer.tsx [hunk 1]
```diff
@@ -1,17 +1,154 @@
...
+
+export const ActorAboutContainer: React.FC<Props> = ({ actors, actor, code }) => {
+  const router = useRouter();
+  const { isLight } = useThemeContext();
+  const [isEnabled] = useFlagsActive();
+  const table834 = useMediaQuery(lightTheme.breakpoints.between('table_834', 'desktop_1194'));
+  const phone = useMediaQuery(lightTheme.breakpoints.between('table_375', 'table_834'));
+  const LessPhone = useMediaQuery(lightTheme.breakpoints.down('table_375'));
+  const { queryStrings } = useActorAbout(router.query);
+  console.log({ router });
```
Comment (YasielCabrera) [inline src/stories/containers/ActorsAbout/ActorAboutContainer.tsx:36]: is this needed?

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/stories/containers/ActorsAbout/ActorAboutContainer.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/stories/containers/ActorsAbout/ActorAboutContainer.tsx:36]: No, I delete it

File: src/stories/containers/ActorsAbout/ActorAboutContainer.tsx [hunk 2]
```diff
@@ -1,17 +1,154 @@
...
-  console.log('actors', actor);
+
+export const ActorAboutContainer: React.FC<Props> = ({ actors, actor, code }) => {
+  const router = useRouter();
+  const { isLight } = useThemeContext();
+  const [isEnabled] = useFlagsActive();
+  const table834 = useMediaQuery(lightTheme.breakpoints.between('table_834', 'desktop_1194'));
+  const phone = useMediaQuery(lightTheme.breakpoints.between('table_375', 'table_834'));
+  const LessPhone = useMediaQuery(lightTheme.breakpoints.down('table_375'));
+  const { queryStrings } = useActorAbout(router.query);
```
[the remark this answers is not in the document]
Comment (alejandrocabriales) [reply src/stories/containers/ActorsAbout/ActorAboutContainer.tsx:35]: you are rigth

**Review by YasielCabrera — APPROVED** [review]
LGTM

**Pull request merged**